server/views/components/instructions/template.njk

```diff
@@ -1,47 +1,42 @@
 {% from "govuk/components/textarea/macro.njk" import govukTextarea %}
 {% if params.inGridRow %}
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-{% endif %}
-{% if params.errorMessage %}
-    <div class="govuk-form-group govuk-form-group--error">
-{% else %}
-    <div class="govuk-form-group">
-{% endif %}
-      <h2 class="govuk-heading-l">
-            {{ params.titleText | safe if params.titleText else "Allocation instructions" }}
-      </h2>
-      <label class="govuk-label govuk-label--l" for="instructions">
-
-        {{ params.html | safe if params.html}}
-
-            <div id="more-detail-hint" class="govuk-hint">
-              {{ params.hintText | safe if params.hintText else "Enter notes for the probation practitioner, which will be sent in the allocation email. You’ll still need to create a Management Oversight contact in NDelius. Do not include links in the notes." }}
-            </div>
-
-{% if params.errorMessage %}
-      <p id="instructions-error" class="govuk-error-message">
-        <span class="govuk-visually-hidden">Error:</span> {{ params.errorMessage.text | safe if params.errorMessage.text }}
-      </p>
-              <textarea class="govuk-textarea" data-crn="{{ params.crn }}" data-conviction-number="{{ params.convictionNumber }}"  id="instructions" name="instructions" rows="{{ params.rows if params.rows else 5}}" aria-describedby="instuctions-hint instructions-error"></textarea>
-
-{% else %}
-              <textarea class="govuk-textarea" data-crn="{{ params.crn }}" data-conviction-number="{{ params.convictionNumber }}"  id="instructions" name="instructions" rows="{{ params.rows if params.rows else 5}}" aria-describedby="instuctions-hint"></textarea>
-
-{% endif %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+    {% endif %}
```

Comment:
Some weird indentation here

Reply:
Agreed, it looks odd but that's because the nunjucks directives are indenting along with the HTML markup and the lines above are opening tags with no closing ones. Ideally I'd like to fix this but I think the way to do that is to have a more granular component structure but that would need to be part of a wider effort